releases/latest/postgresql-bundle.yaml

```diff
@@ -0,0 +1,31 @@
+series: focal
+applications:
+  pgbouncer:
+    charm: pgbouncer
+    channel: edge
+    revision: 7
+    num_units: 1
+    to:
+    - "1"
+    constraints: arch=amd64
+  postgresql:
+    charm: cs:postgresql
+    channel: edge
+    revision: 254
+    resources:
+      patroni: 1
+    num_units: 1
+    to:
+    - "0"
+    constraints: arch=amd64
+    storage:
+      pgdata: rootfs,1,1024M
+    trust: true
```

Comment:
I think trust is needed only on k8s (afaik it is for authorising the charm to connect to the k8s api).